rllib/policy/eager_tf_policy.py

```diff
@@ -313,10 +313,7 @@ def compute_actions(self,
             self._is_training = False
             self._state_in = state_batches
 
-            if tf.executing_eagerly():
-                n = len(obs_batch)
-            else:
-                n = obs_batch.shape[0]
+            n = len(obs_batch)
```

Comment:
Doesn't len() not work in graph mode or am I missing something?

Reply:
I'm not sure. I just thought: Why would we check on tf.executing_eagerly() if in an eager policy? Are we not never in graph mode in that part of the code?

Reply:
Oh hmm true. I think if tracing is on then sometimes you might end up in graph mode but not sure.

Reply:
Yeah, it broke :) I'll leave it for now.